### PR TITLE
{vis}[GCCcore/7.3.0] jbigkit 2.1

### PR DESCRIPTION
**Pull request by fizwit**
(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyconfigs/j/jbigkit/jbigkit-2.1_fix-libpath.patch [hunk 1]
```diff
@@ -0,0 +1,26 @@
+build process does not find its own libraries
+
```
Comment (akesandgren) [inline easybuild/easyconfigs/j/jbigkit/jbigkit-2.1_fix-libpath.patch]: Please add some Author information here. And don't forget to update the checksum too

File: easybuild/easyconfigs/j/jbigkit/jbigkit-2.1_fix-libpath.patch [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+diff -ruN jbigkit-2.1.orig/Makefile jbigkit-2.1/Makefile
+--- jbigkit-2.1.orig/Makefile	2014-03-27 11:47:15.000000000 -0700
++++ jbigkit-2.1/Makefile	2019-03-21 12:05:55.442478000 -0700
+@@ -4,7 +4,7 @@
+ CC = gcc
+ 
+ # Options for the compiler: A high optimization level is suggested
+-CFLAGS = -O2 -W -Wno-unused-result
++CFLAGS = -O2 -W -Wno-unused-result -I../libjbig
```
Comment (akesandgren) [inline easybuild/easyconfigs/j/jbigkit/jbigkit-2.1_fix-libpath.patch]: Preferably it should pick up the CFLAGS settings from EasyBuild, i.e. CFLAGS += -W -Wno-unused...
and not set -O2

File: easybuild/easyconfigs/j/jbigkit/jbigkit-2.1_fix-libpath.patch [hunk 3]
```diff
@@ -0,0 +1,26 @@
+build process does not find its own libraries
+
+diff -ruN jbigkit-2.1.orig/Makefile jbigkit-2.1/Makefile
+--- jbigkit-2.1.orig/Makefile	2014-03-27 11:47:15.000000000 -0700
++++ jbigkit-2.1/Makefile	2019-03-21 12:05:55.442478000 -0700
+@@ -4,7 +4,7 @@
+ CC = gcc
```
Comment (akesandgren) [inline easybuild/easyconfigs/j/jbigkit/jbigkit-2.1_fix-libpath.patch]: This should either pick up $CC from the env explicitly or be commented out so it picks it up automatically.

File: easybuild/easyconfigs/j/jbigkit/jbigkit-2.1_fix-libpath.patch [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+ export CC CFLAGS
+diff -ruN jbigkit-2.1.orig/pbmtools/Makefile jbigkit-2.1/pbmtools/Makefile
+--- jbigkit-2.1.orig/pbmtools/Makefile	2014-03-27 11:47:15.000000000 -0700
++++ jbigkit-2.1/pbmtools/Makefile	2019-03-21 11:51:23.338623000 -0700
+@@ -4,7 +4,7 @@
+ CC = gcc
+ 
+ # Options for the compiler
+-CFLAGS = -g -O -W -Wall -Wno-unused-result -ansi -pedantic # --coverage
++CFLAGS = -I../libjbig -g -O -W -Wall -Wno-unused-result -ansi -pedantic # --coverage
```
Comment (akesandgren) [inline easybuild/easyconfigs/j/jbigkit/jbigkit-2.1_fix-libpath.patch]: Same here for CC and CFLAGS, use the settings from EasyBuild

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2658.swalot.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/6890a66990b8b345c20935669b99e82c for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2053.delcatty.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/c58e7055d45e59fd3f8502feb7d315dc for a full test report.

**Comment by boegel** [comment]
@fizwit Please remove `easybuild/easyconfigs/j/jbigkit/jbigkit-2.1_fix-libpath.patch`, it's no longer needed?

```
eb --update-pr 8012 :easybuild/easyconfigs/j/jbigkit/jbigkit-2.1_fix-libpath.patch --pr-commit-msg "remove unused patch for jbigkit"

**Comment by fizwit** [comment]
I am closing and reopening, to remove the patch.

**Pull request closed without merging**